package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@govuk-one-login/di-ipv-cri-common-express",
-  "version": "6.4.0",
+  "version": "6.5.0",
```

Comment:
Do we think this is a patch (bug fix) rather than a minor version?

Reply:
making this a minor version was decided in a chat with robin.

Reply:
@Caitlin-cooling / @liamStockwellGDS You're both suggesting a patch release, so I'm happy to drop it down to that :)